application/cmd/integrum/main.go

```diff
@@ -0,0 +1,260 @@
+	cfg.Formatter = "text"
+	if verbose {
+		cfg.Level = insolar.DebugLevel.String()
+	} else {
+		cfg.Level = insolar.WarnLevel.String()
+	}
+
+	defaultCfg := logadapter.DefaultLoggerSettings()
+	defaultCfg.Instruments.CallerMode = insolar.NoCallerField
+	defaultCfg.Instruments.MetricsMode = insolar.NoLogMetrics
+	logger, _ := log.NewLogExt(cfg, defaultCfg, 0)
+	ctx = inslogger.SetLogger(ctx, logger)
+	log.SetGlobalLogger(logger)
+
+	return ctx
+}
+
+func createUserConfig(privateKey crypto.PrivateKey) (*requester.UserConfigJSON, error) {
+	privateKeyBytes, err := secrets.ExportPrivateKeyPEM(privateKey)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to export private key")
+	}
+	privateKeyStr := string(privateKeyBytes)
+
+	publicKey, err := secrets.ExportPublicKeyPEM(secrets.ExtractPublicKey(privateKey))
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to extract public key")
+	}
+	publicKeyStr := string(publicKey)
+
+	return requester.CreateUserConfig("", privateKeyStr, publicKeyStr)
+}
+
+// requireUrlArg returns an error if there is not url args.
+func requireUrlArg() cobra.PositionalArgs {
+	return func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
+			return errors.New("The program required url as an argument")
+		}
+		return nil
+	}
+}
+
+func DepositCreatorCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "integrum <insolar_endpoint>",
+		Short:   ApplicationShortDescription,
+		Args:    requireUrlArg(),
+		Example: "./integrum http://localhost:19001/admin-api/rpc -p http://localhost:19101/api/rpc -k 'Path to a key pair' -u postgresql://localhost:5432/postgres?sslmode=disable -v",
+		RunE: func(_ *cobra.Command, args []string) error {
+			// no need to check args size because of requireUrlArg
+			apiURL = args[0]
+			if len(apiURL) > 0 {
+				ok, err := isUrl(apiURL)
+				if !ok {
+					return errors.Wrap(err, "URL parameter is incorrect.")
+				}
+			}
+
+			e := verifyParams()
+			if e != nil {
+				return e
+			}
+			ctx := getContextWithLogger()
+			requester.SetVerbose(verbose)
+
+			userConfig, e := createUserConfig(memberPrivateKey)
+			if e != nil {
+				return e
+			}
+
+			reference, e := getMemberReference(ctx, userConfig)
+			if e != nil {
+				return errors.Wrap(e, "fail to get from member reference")
+			}
+			userConfig.Caller = reference
+
+			opt, err := pg.ParseURL(dbURL)
+			if err != nil {
+				panic(err)
+			}
+
+			db := pg.Connect(opt)
+			defer db.Close()
+			var members []DepositMember
+			rows, err := db.Query(&members, "SELECT d.deposit_ref, d.member_ref FROM deposits d "+
+				"WHERE d.eth_hash != 'genesis_deposit' AND d.eth_hash != 'genesis_deposit2' "+
+				"AND d.eth_hash NOT IN (SELECT d1.eth_hash FROM deposits d1 WHERE d1.eth_hash like '%_2');")
+			if err != nil {
+				panic(err)
+			}
+
+			if rows.RowsAffected() == 0 {
+				return errors.New("Members without second deposit not found")
+			}
+			fmt.Println(rows.RowsAffected())
```

Comment:
Looks like the query doesn't work as expected.

When running a query on test data, I got this
![image](https://user-images.githubusercontent.com/19386609/91309065-d4e50900-e7b8-11ea-8348-bb506b6966bb.png)


```suggestion
			err = db.Model().
				Table("deposits").
				Column("deposit_ref", "member_ref").
				Where("status = 'confirmed'").
				Where("eth_hash != 'genesis_deposit'").
				Where("eth_hash != 'genesis_deposit2'").
				Where("eth_hash not like E'%\\_2'").
				Where(`not exists (
					select 1 
					from deposits as inn
					where inn.eth_hash = concat("deposits".eth_hash, '_2')
				)`).
				Select(&members)
			if err != nil {
				panic(err)
			}

			if len(members) == 0 {
				return errors.New("Members without second deposit not found")
			}
			fmt.Println(len(members))
```